homeassistant/components/fritz/common.py

```diff
@@ -194,26 +245,25 @@ async def service_fritzbox(self, service: str) -> None:
             raise HomeAssistantError("Service not supported") from ex
 
 
+@dataclass
 class FritzData:
     """Storage class for platform global data."""
 
-    def __init__(self) -> None:
-        """Initialize the data."""
-        self.tracked: dict = {}
+    tracked: dict = field(default_factory=dict)
 
 
 class FritzDevice:
     """FritzScanner device."""
 
-    def __init__(self, mac, name=None):
+    def __init__(self, mac: str, name: str) -> None:
```

Comment:
Before `name` had a default value of `None`, is it needed?

Reply:
No more needed with latest rework